pkgs/by-name/ap/apple-sdk/common/plists.nix

```diff
@@ -0,0 +1,307 @@
+{
+  lib,
+  stdenvNoCC,
+  xcodePlatform,
+}:
+
+let
+  inherit (lib.generators) toPlist;
+
+  Info = {
+    CFBundleIdentifier = "com.apple.platform.${lib.toLower xcodePlatform}";
+    Type = "Platform";
+    Name = lib.toLower xcodePlatform;
+  };
+
+  # These files are all based off of Xcode spec fies found in
+  # /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Xcode/PrivatePlugIns/IDEOSXSupportCore.ideplugin/Contents/Resources.
```

Comment:
This file was moved [from xcbuild][1] because these definitions need to be in the platform definition in the SDK for xcbuild to work. If Apple introduces a new Xcode project format that xcbuild doesn’t understand, divergence is probably the least of our problems.

[1]: https://github.com/NixOS/nixpkgs/blob/51299433482c463e36fc8997c1b19f40fa7374c5/pkgs/development/tools/xcbuild/platforms.nix.